js/works.js

```diff
@@ -62,3 +62,10 @@ function writeItem(){
 };
 
 writeItem();
+$('nav').on('click', function(e){
+  var $name = ($(e.target).data('name'));
```

Comment:
@ptrompeter This chunk of code works but can you explain your reasoning behind using the passed in event reference and why you attached that to a target?  Not nitpicking, just curious.